frontend/encore/simple-example.rst

```diff
@@ -70,6 +70,11 @@ To build the assets, use the ``encore`` executable:
     # compile assets, but also minify & optimize them
     $ ./node_modules/.bin/encore production
 
+    # shorter version of the above 3 commands
+    $ yarn run encore dev
+    $ yarn run encore dev -- --watch
```

Comment:
It is. Adding the first one tells yarn to pass the rest of the arguments to Encore instead of directly using them ([ref.](https://yarnpkg.com/lang/en/docs/cli/run/#toc-yarn-run-script)).